scripts/remove

```diff
@@ -33,7 +33,6 @@ ynh_remove_logrotate
 ynh_remove_nginx_config
 
 # Remove the log file
-ynh_secure_remove --file="/var/log/$app"
```

Comment:
The previous line should be removed as well (/cc @alexAubin)

Reply:
(No need to ping me specifically, I'm not the author of this automatic patch...)